### DISCUSSION TITLE
How to change column width ?

### DISCUSSION THREAD

**Discussion by Uhasith** [Power-Components/livewire-powergrid#1659] [Q&A]
I want to apply a `max-w-[200px] truncate` class to the column to prevent long text from taking up too much viewport space. I tried the method below, and while it works, the UI isn’t very great. The adjacent columns seem to be too close to each other.

```
  Column::add()
                ->title('Product Name')
                ->field('name')->searchable()->fixedOnResponsive()->bodyAttribute('max-w-[200px] text-left truncate'),
```

**Comment by Uhasith** [comment]
can I set a max column width to a specific column in the table ?

**Reply by KVasquesMoviaUTN** [reply]
Hello, I was having the same issue. I got it to work. In V6 you need to do something like this:

  public function fields(): PowerGridFields
  {
    return PowerGrid::fields()
        ->add('titulo_blade', function ($row) {
          return Blade::render(
            '<div style="width: 600px;">{{ $titulo }}</div>',
            [
              'id' => $row->ni_id,
              'titulo' => $row->ni_titulo,
            ]
          );
        });
       }
       
public function columns(): array
  {
    return [
      Column::make('Titulo', 'titulo_blade')->template(),
          ];
  }
  
By adding ->template() you can overwrite certain aspects ot the html, and then you just redefine it in your fields() by using render() method, I hope it works!
